modules/scalaz/build.sbt

```diff
@@ -0,0 +1,16 @@
+name := "pureconfig-scalaz"
+
+libraryDependencies ++= Seq(
+  "org.scalaz" %% "scalaz-core" % "7.2.27",
+  "org.scalaz" %% "scalaz-scalacheck-binding" % "7.2.27-scalacheck-1.13"
```

Comment:
Shouldn't this be a `% "test"` dependency?

Reply:
Yep, missed that. FIxed in ef62dd9.